scripts/index.js

```diff
@@ -0,0 +1,61 @@
+let tempValue = document.getElementById("tempValue");
+let temperature = 63;
+let landscape = document.getElementById("landscape")
+let skySelect = document.getElementById("skySelect")
+let skyElement = document.getElementById("sky")
+
+let increaseTempControl = document.getElementById("increaseTempControl");
+increaseTempControl.addEventListener("click", function () {
```

Comment:
You repeat a lot of logic between `increaseTempControl` and `decreaseTempControl` a helper function could DRY up your code.